assets/js/controllers/faq.controller.js

```diff
@@ -2,7 +2,11 @@ angular
   .module('walletApp')
   .controller('faqCtrl', faqCtrl);
 
-function faqCtrl ($scope, faq) {
+function faqCtrl ($scope, faq, buyStatus) {
   $scope.questions = faq.questions.map(q => angular.merge({ displayed: false }, q));
   $scope.toggle = (q) => { q.displayed = !q.displayed; };
+
+  buyStatus.canBuy().then((canBuy) => {
+    !buyStatus.userHasAccount() && !canBuy && ($scope.questions[0] = {name: 'CAN_I_BUY_UNINVITED'});
```

Comment:
Need to remember that this is here if we ever change the first FAQ from `CAN_I_BUY`.